lib/model/bloc/transactions/TransactionToRecordTransactionViewModel.dart

```diff
@@ -54,7 +54,7 @@ class TransactionToRecordTransactionViewModel
 
   List<RecordTransactionListItemViewModel> _actions({TransactionEntity from}) {
     final timestamp = (from == null) ? DateTime.now() : from.timestamp;
-    final description = (from == null) ? "" : from.description;
+    final description = (from == null) ? "" : (from.description ?? "");
```

Comment:
I think this could be simplified to 
from?.description ?? ""

Reply:
👍

Reply:
done